src/utils/getIssueAntdPath.ts

```diff
@@ -17,21 +17,37 @@ const serializePath = (
   const baseSchema = getSchemaBaseSchema(schema);
   if (isZodObject(baseSchema)) {
     antdPath.push(subPath);
-    const subPathBaseSchema = getSchemaBaseSchema(baseSchema.shape[subPath]);
 
-    return isZodObject(subPathBaseSchema)
-      ? serializePath(subPathBaseSchema, zodPath, antdPath)
-      : concatAntdPath(antdPath);
+    const replacedPath = subPath.replace(/\.\d+/, "");
```

Comment:
Similar to a comment bellow, this should be moved to a function called `prepareZodPath(antdPath: string)`